api/channel.go

```diff
@@ -877,6 +885,8 @@ func updateLastViewedAt(c *Context, w http.ResponseWriter, r *http.Request) {
 		active = true
 	}
 
+	channelName := data["channel_name"].(string)
```

Comment:
Also, why is this done by channel name? I assume it's for the town square, but that'll break it if someone renames a channel